### DISCUSSION TITLE
The commit belongs to the wrong time

### DISCUSSION THREAD

**Discussion by sy-records** [community/community#73285] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

I merged two PRs today using `Squash and merge`.

<img width="307" alt="image" src="https://github.com/community/community/assets/33931153/ef1086cb-9abd-4a13-a281-db62afcbe61f">

But they were both commited yesterday.
<img width="594" alt="image" src="https://github.com/community/community/assets/33931153/20dd4321-4cae-4151-89dc-4e9dc4993c4b">

see https://github.com/php/doc-zh/commits/master/

https://github.com/php/doc-zh/pull/809
https://github.com/php/doc-zh/pull/808

**Comment by GigaSmurf** [comment]
The new commit created by GitHub would typically use the timestamp of the most recent commit in the PR. The timestamp displayed on the commits list would show when the squash and merge action was performed, not necessarily the timestamp of the original commits probably.

**Reply by sy-records** [reply]
yeah, I know. 

But now the time in the list is wrong, it's earlier than the time I merged.

**Comment by sy-records** [comment, marked as answer]
see https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/troubleshooting-commits-on-your-timeline#viewing-missing-commit-details-from-commits-in-your-timeline

use `AuthorDate` not `CommitDate`.
